web/app/js/components/DeploymentDetail.jsx

```diff
@@ -160,55 +148,22 @@ export default class DeploymentDetail extends React.Component {
   }
 
   renderMidsection() {
```

Comment:
TIOLI the Deployment Details section is quite awkward on a row by itself, and it was slated to be removed anyway. I'd just remove all of `renderMidsection()` (or I will, in a follow up branch; there's more things that we're taking out too).